src/NzbDrone.Core/ImportLists/Trakt/Popular/TraktPopularRequestGenerator.cs

```diff
@@ -52,6 +52,18 @@ private IEnumerable<ImportListRequest> GetSeriesRequest()
                 case (int)TraktPopularListType.TopWatchedByAllTime:
                     link += "/shows/watched/all" + filtersAndLimit;
                     break;
+                case (int)TraktPopularListType.RecommendedByWeek:
```

Comment:
I think a small refactor here will go a long way, we don't need to add `filtersAndLimit` to each case, we can add it at the end, each case would change to something like: `link += "/shows/recommended/weekly"` and the outside the switch, `link += filtersAndLimit`